taf/tests/test_repository_tool.py

```diff
@@ -90,25 +91,37 @@ def test_update_timestamp_wrong_key(repositories, snapshot_key):
 
 def test_update_targets_from_keystore_valid_key(repositories, targets_key):
     taf_happy_path = repositories["test-happy-path"]
-    start_date = datetime.datetime.now()
-    interval = 1
-    expected_expiration_date = to_tuf_datetime_format(start_date, interval)
+    targets_path = Path(taf_happy_path.targets_path)
+    repositories_json_path = targets_path / "repositories.json"
+
+    branch_id = "14e81cd1-0050-43aa-9e2c-e34fffa6f517"
+    target_commit_sha = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+    repositories_json_old = repositories_json_path.read_text()
+
+    targets_data = {
+        "branch": {"target": branch_id},
+        "dummy/target_dummy_repo": {"target": {"commit": target_commit_sha}},
+        "capstone": {},
```

Comment:
Since branch and capstone are specific to our implementation, I think that we should use random files for testing purposes.